### ISSUE TITLE
Virtual `STOP` do not pass circuits

### ISSUE THREAD

**Issue by adria0** [privacy-scaling-explorations/zkevm-circuits#456]
It seems that virtual `STOP` is not passing the circuits:

explicit `STOP` works:

```
let bytecode = bytecode! {
    GAS
    STOP
};
assert_eq!(run_test_circuits(bytecode), Ok(()));
```

but virtual `STOP` fails:

```
let bytecode = bytecode! {
    GAS
};
assert_eq!(run_test_circuits(bytecode), Ok(()));
```

**Comment by han0110** [comment]
Yes, currently bytecodes need to have a explicitly halting opcode to pass EVM circuit. It will be implemented in #419 (will update the PR for review as soon as possible).

**Comment by adria0** [comment]
:+1: thanks @han0110

**Comment by han0110** [comment]
Closed by #419
